backend/apps/slack/management/commands/slack_sync_messages.py

```
@@ -0,0 +1,158 @@
+                    channel=conversation.slack_channel_id,
+                    cursor=cursor,
+                    limit=batch_size,
+                    latest=latest,
+                )
+                self._handle_slack_response(response, "conversations_history")
+                messages_data = response.get("messages", [])
+
+                messages = []
+                for message_data in messages_data:
+                    if message := self._create_message_from_data(message_data, conversation):
+                        messages.append(message)
```

Ruff (0.11.9)

Use a list comprehension to create a transformed list

(PERF401)